### PR TITLE
Issue #179 - Add custom ticks for values in thousands

### PR DESCRIPTION
**Pull request by aimenbatool**

### CODE REVIEW

**Comment by djmitche** [comment]
From #179, it sounds like we only want to use `K` when the last three digits are `0`, because some metrics only give results rounded to the nearest thousand.

As this patch is written, a metric with values like 1028, 1057, 1109 will show up as "1K, 1K, 1K", losing useful precision.

It would be really great if the "K" conversion would only happen if *all* values in the chart ended with three `0`'s, but that might not be practical.

**Comment by armenzg** [comment]
We are only focusing on the ticks of the Y labels rather than the tooltip.

**Comment by aimenbatool** [comment]
> We are only focusing on the ticks of the Y labels rather than the tooltip.

I believed [it is](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/238/files#diff-fd4ef18e4ab918f44b2c808156f5e496R43) the configuration for Y label not for tooltip. All configuration for tooltips comes under [tooltip here.](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/238/files#diff-fd4ef18e4ab918f44b2c808156f5e496R40)

Correct me if I am wrong.

File: src/containers/RedashContainer/index.jsx [hunk 1]
```diff
@@ -47,6 +48,9 @@ class RedashContainer extends Component {
           },
         },
       },
+      ticksCallback: (value) => {
+        return value > 999 ? `${(value / 1000)}k` : value;
```
[the remark this answers is not in the document]
Comment (aimenbatool) [reply src/containers/RedashContainer/index.jsx]: Updated :)

**Pull request merged**